tf2onnx/utils.py

```diff
@@ -121,17 +121,20 @@ def tf_to_onnx_tensor(tensor, name=""):
     new_type = TF_TO_ONNX_DTYPE[tensor.dtype]
     tdim = tensor.tensor_shape.dim
     dims = [d.size for d in tdim]
-    # FIXME: something is fishy here
-    if dims == [0]:
-        dims = [1]
     is_raw, data = get_tf_tensor_data(tensor)
+    # data is [], i.e., empty array
```

Comment:
please add more comments with examples, otherwise, it is hard for others to understand.